docs/src/main/tut/datatypes/validated.md

```diff
@@ -12,7 +12,298 @@ Response comes back saying your username can't have dashes in it, so you make so
 have special characters either. Change, resubmit. Passwords need to have at least one capital letter. Change,
 resubmit. Password needs to have at least one number.
 
-Or perhaps you're reading from a configuration file. One could imagine the configuration library you're using returns
+It would be nice to have all of these errors be reported simultaneously. That the username can't have dashes can
+be validated separately from it not having special characters, as well as from the password needing to have certain
+requirements. A misspelled (or missing) field in a config can be validated separately from another field not being
+well-formed.
+
+Enter `Validated`.
+
+## A first approach
+
+You'll note firsthand that `Validated` is very similar to `Either` because it also has two possible values: errors on the left side or successful computations on the right side.
+
+Signature of the structure is as follows:
+
+```scala
+sealed abstract class Validated[+E, +A] extends Product with Serializable {
+	// Implementation elided
```

Comment:
Tiny nitpick, but can we reduce the indentation here?

Reply:
Is there a standard used by cats? Spaces, tabs?

Reply:
two spaces is standard :)

Reply:
Got it! I'll fit it in all the code snippets.